### ISSUE TITLE
Contao 4.4.49: Uncaught Error: Call to undefined method Symfony\Component\Config\Definition\Builder\TreeBuilder::getRootNode()

### ISSUE THREAD

**Issue by Defcon0** [codefog/tags-bundle#20]
Hello,

I just tried to install the tags bundle within a contao 4.4.49 installation with the following error:

`Uncaught Error: Call to undefined method Symfony\Component\Config\Definition\Builder\TreeBuilder::getRootNode()`

in `vendor/codefog/tags-bundle/src/DependencyInjection/Configuration.php`.

It can easily be fixed using the following code instead:

```
public function getConfigTreeBuilder()
    {
        $treeBuilder = new TreeBuilder();
        $treeBuilder->root('codefog_tags')
            ->children()
                ->arrayNode('managers')
                    ->useAttributeAsKey('name')
                    ->arrayPrototype()
                    ->children()
                        ->variableNode('source')
                            ->isRequired()
                            ->validate()
                                ->ifString()
                                ->then(
                                    static function (string $value): array {
                                        return [$value];
                                    }
                                )
                            ->end()
                        ->end()
                        ->scalarNode('service')->defaultValue('codefog_tags.default_manager')->end()
                        ->scalarNode('alias')->defaultNull()->end()
                    ->end()
                ->end()
            ->end()
        ;

        return $treeBuilder;
    }
```

Could you please insert the code in the repo? Thanks!

**Comment by qzminski** [comment]
Should be fixed in 3.0.3 that I just released.

**Comment by Defcon0** [comment]
Works! Thank you!
